apps/explorer/src/components/module/PkgModulesWrapper.tsx

```diff
@@ -250,7 +250,10 @@ function PkgModuleViewWrapper({ id, modules, splitPanelOrientation }: Props) {
                     />
                 </div>
             ) : (
-                <>{bytecodeContent}</>
+                bytecodeContent &&
```

Comment:
bytecodeContent is always an array if I'm looking at this right, so I don't think we need this truthy check